docs/pages/debugging.md

```diff
@@ -0,0 +1,26 @@
+Debugging ArrayFire Issues {#debugging}
+===============================================================================
+
+Some general advice for debugging mysterious issues with ArrayFire follows.
+
+First, try running with additional debugging information:
+
+    export AF_PRINT_ERRORS=1           #Make exceptions more helpful
+    export AF_TRACE=all                #Print all the stuff
+    #export AF_TRACE=mem               #Print memory management allocation, free, and garbage collection information
+    #export AF_TRACE=unified           #Print unified backend dynamic loading information
+    #export AF_TRACE=mem,unified       #Print both of the above
```

Comment:
We shouldn't mention each and every possible value of each environment variable here, that should be limited  to the respective environment variable documentation in the "Configuring ArrayFire Environment" page.

`export AF_TRACE=all` is sufficient for explanation here and you can hyperlink the respective environment variable documentation here.

Reply:
Also, a bullet list of variables(not all possible values) available instead of code section, this will also fit perfectly with the idea of just hinting the variable and hyperlinking the related docs.

Note: Just in case - you can see the timing.md to check for syntax of lists.

Reply:
I've moved to a list. Not sure about the internal linkages.